match/lib/match/nuke.rb

```diff
@@ -13,7 +13,7 @@ def run(params, type: nil)
 
       params[:workspace] = GitHelper.clone(params[:git_url], params[:shallow_clone], skip_docs: params[:skip_docs], branch: params[:git_branch])
 
-      had_app_identifier = self.params[:app_identifier]
+      had_app_identifier = self.params.fetch(:app_identifier, ask: false)
```

Comment:
i think its intended to disable the auto ask for the user. so in case `app_identifier` is not set - set `had_app_identifier` to nil instead of prompting user to set a app_identifier